util/src/main/i18n/templates/misc.properties

```diff
@@ -54,6 +54,8 @@ misc.timeAgo = {0} ago
 # {0} = duration (e.g. "1:00" for 1 minute left, "15:25" for 15 minutes and 25 seconds left)
 misc.timeRemaining = Time Remaining: {0}
 
+misc.overtime = Overtime {0}
```

Comment:
`Overtime: {0}`

Reply:
We tested this format and it did not look the best, due to overtime being in bold, and that causing the `:` to also be bolded

![image](https://user-images.githubusercontent.com/11789291/90428473-56a3ab00-e0c4-11ea-9cb2-d1bf14184006.png)
![image](https://user-images.githubusercontent.com/11789291/90428639-979bbf80-e0c4-11ea-897c-1ad01d1bbb48.png)